sn_client/src/wallet.rs

```diff
@@ -663,6 +663,30 @@ impl WalletClient {
 impl Client {
     /// Send spend requests to the network.
     /// This can optionally verify the spends have been correctly stored before returning
+    ///
+    /// # Arguments
+    /// * spend_requests - [Iterator]<[SignedSpend]>
+    /// * verify_store - Boolean. Set to true for mandatory verification via a GET request through a Spend on the network.
```

Comment:
```suggestion
    /// * verify_store - [bool]. Set to true for mandatory verification via a GET request through a Spend on the network.
```

Reply:
I've had trouble with bool. for [bool] it doesn't point anywhere on the IDE or the docs when you hover over it for help, so we need to link it to something. Links go in the curly brackets after the square ones, so the result would look like this:

[bool](link to bool)

problem there is that I can not find a good link to the bool, and it's not easy to find online. 
I tried two options:

[bool](core::primitive::bool)
and
[bool](std::primitive::bool)

but no luck. It does however link to the bool struct if you use either of these:

[bool](core::primitive)
and
[bool](std::primitive)

but that didn't feel right so I just left it at 'Boolean'. I figure if you don't know what a Boolean is, maybe you need more help than from the docs - however the right link to wherever the Boolean is stored in Rust is useful. 

So you are on the right track for pointing this out, but I think we can keep an eye out for a better solution. 
What do you think?

Reply:
Ah. I use vscode and `[bool]` points to https://doc.rust-lang.org/nightly/core/primitive.bool.html
I assume it is because bool might already be in scope? 

Anyway, I agree with your point. This is a very trivial thing, will mark it as 'resolved'.